indigo/distribution.yaml

```diff
@@ -8732,6 +8732,11 @@ repositories:
       url: https://github.com/WPI-RAIL/ros_ethernet_rmp.git
       version: develop
     status: maintained
+  ros_in_hand_scanner:
+    doc:
+      type: git
+      url: https://github.com/RodBelaFarin/ros_in_hand_scanner.git
+      version: developed
```

Comment:
I am pretty sorry. Yes, I do mean master. Do I have to make any changes
now?
Am 03.02.2016 5:39 nachm. schrieb "Dirk Thomas" <notifications@github.com>:

> In indigo/distribution.yaml
> <https://github.com/ros/rosdistro/pull/10403#discussion_r51747962>:
>
> > @@ -8732,6 +8732,11 @@ repositories:
> >        url: https://github.com/WPI-RAIL/ros_ethernet_rmp.git
> >        version: develop
> >      status: maintained
> > +  ros_in_hand_scanner:
> > +    doc:
> > +      type: git
> > +      url: https://github.com/RodBelaFarin/ros_in_hand_scanner.git
> > +      version: developed
>
> There is no such branch in your repo. Do you mean master?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros/rosdistro/pull/10403/files#r51747962>.
>